app/src/main/java/org/mozilla/fenix/tabstray/browser/BrowserTrayInteractor.kt

```diff
@@ -65,14 +76,21 @@ class DefaultBrowserTrayInteractor(
      * See [SelectionInteractor.open]
      */
     override fun open(item: Tab) {
```

Comment:
This is the `open` method from the `SelectionInteractor` which we have to implement even if we want only the select/deselect functionality.

Reply:
SelectionInteractor was originally used for all the library lists, so handling the "open" case made sense. We moved this out to a higher package level because it was pretty convenient to re-use that same API in the tabs tray as well. 🙂